src/software/ai/navigator/navigator.cpp

```diff
@@ -319,7 +289,7 @@ std::vector<Obstacle> Navigator::createCurrentObstacles(
     return obstacles;
 }
 
-double Navigator::getCloseToEnemyObstacleFactor(Point &p)
+double Navigator::getCloseToEnemyObstacleFactor(Point p)
```

Comment:
Why is the argument no longer being passed in as a reference?

Reply:
It wasn't a const reference which conflicted with const points being passed in. Fixed